go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bitrise-steplib/steps-xcode-archive
 
-go 1.22
+go 1.23.0
```

Comment:
Can we avoid bumping the Go version? It's the usual preinstalled version problem.

Reply:
I do not think it would matter as the toolchain is already set to 1.23.
Some dependencies require 1.23 it is hard to avoid.

Reply:
Oh, good point, I agree